packages/oruga-next/src/components/select/Select.vue

```diff
@@ -300,6 +304,12 @@ const rootClasses = defineClasses(
         null,
         computed(() => props.expanded),
     ],
+    [
+        "rootVariantClass",
+        "o-ctrl-sel--",
+        computed(() => statusVariant.value || props.variant),
```

Comment:
Could you give me a link why the variant is needed at the root element here?

Reply:
Sure, this is just how Bulma works. See https://github.com/jgthms/bulma/blob/master/sass/form/select.sass#L44

Reply:
Hmm ok its fine but when I look at all the classes of the select component I think I should refactor them and move some up to the root container element....